src/org/omegat/gui/theme/DelegatingLookAndFeel.java

```diff
@@ -78,6 +82,95 @@ public DelegatingLookAndFeel() throws Exception {
         systemLookAndFeel = DesktopSupport.getSupport().createLookAndFeel(systemLafName);
     }
 
+    /**
```

Comment:
I think your overall idea is good:

1. We should only apply the default colors for the default themes
2. But it should be easy for third-party themes to make use of the default colors as well

But I don't think this code belongs in `DelegatingLookAndFeel`. I think it belongs in one of two places:

- An `OmegaTBaseTheme` that inherits from `DelegatingLookAndFeel`, and from which both default themes inherit from, OR
- It stays in `UIDesignManager` and is called statically from the themes

Since these methods are static, I don't think it makes sense to insert them in the class hierarchy; I feel like they make more sense in `UIDesignManager`.

Reply:
> It stays in UIDesignManager and is called statically from the themes

I like it.